pkg/trivyk8s/trivyk8s_test.go

```diff
@@ -87,3 +89,50 @@ func TestFilterResource(t *testing.T) {
 		})
 	}
 }
+
+func TestInitResources(t *testing.T) {
+	tests := []struct {
+		name         string
+		includeKinds []string
+		excludeKinds []string
+		want         []string
+	}{
+		{
+			"scan only pods",
+			[]string{"pods"},
+			nil,
+			[]string{k8s.Pods},
+		},
```

Comment:
nit: Can you add struct member names here? It's a little easier to read.

```suggestion
		{
			name: "scan only pods",
			includeKinds: []string{"pods"},
			excludeKinds: nil,
			want: []string{k8s.Pods},
		},
```

Reply:
done https://github.com/aquasecurity/trivy-kubernetes/pull/395/commits/3cee92f75e565d7ea68352319cd001a4bad720d1